tools/removeVols.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+filter=${1:-random-named}
```

Comment:
This is the only thing that gives me a bit headakes. Imagine you delte all randomly named volumes, the ones from DSpace but also the ones from other services. You shouldn’t use this script if you do not know what that means, but we should also warn you and get an acknowledment that you know how distructive this can be. Volumes are the place for container to store stafe they want to have secured above their lifetime, they potentially contain important data.

Reply:
Changed from a script to a markdown file for reference